README.md

```diff
@@ -1,8 +1,12 @@
 # Autoretrieve
 
-Autoretrieve is a standalone gateway server / Bitswap provider that bridges Filecoin Graphsync content over to Bitswap clients. 
+Autoretrieve is a standalone Graphsync-to-Bitswap proxy server that allows IPFS clients to retrieve data which may be available on the Filecoin network but not on IPFS. 
```

Comment:
> which may be available on the Filecoin network but not on IPFS

How does this work?

Reply:
I think some details about how Autoretreieve advertisements work would be good here @elijaharita

Reply:
added more info

Reply:
The new information looks good, but I'm still confused about data which may be available on the Filecoin network but not on IPFS. Shouldn't all data stored on IPFS?

Reply:
@Kevin-Mok i expanded the first sentence to hopefully clarify a bit

Reply:
What's the difference between a Filecoin and IPFS storage provider?